tests/tpot_tests.py

```diff
@@ -103,8 +103,21 @@ def closing(arg):
 # Set up a small test dataset
 
 pretest_X, pretest_y = make_classification(n_samples=100, n_features=10, random_state=42)
-# Set up pandas DataFrame for testing
 
+# Set up artificial groups
+n_datapoints = pretest_y.shape[0]
+groups = np.random.randint(low=0, high=n_datapoints//2, size=n_datapoints)
+
+# Set up custom cv split generators
+custom_cvs = [
+    model_selection.LeaveOneGroupOut().split(X=pretest_X, y=pretest_y, groups=groups),
+    # model_selection.LeavePGroupsOut(2).split(X=pretest_X, y=pretest_y, groups=groups),
```

Comment:
Ups realized I forgot to uncomment these. I will uncomment and update after others review.

Reply:
Please uncomment it.